release-content/0.16/release-notes/occlusion_culling.md

```diff
@@ -0,0 +1,32 @@
+When it comes to pushing hardware to support larger levels and more detailed meshes, "simply draw less stuff" is a tried-and-true strategy.
+These techniques are collectively referred to as "culling": we can save work whenever we can determine "this doesn't need to be drawn" for cheaper than it would cost to draw the thing despite its pixels later being overwritten completely.
+
+**Occlusion culling** is the idea that we don't need to draw something that's completely blocked by other opaque objects,
+from the perspective of the camera.
+That makes sense! We don't need to draw a car parked on the other side of a building (even if it's within the range used for fustrum culling), and we *never* need to draw all of the teeth inside of a character's closed mouth.
+
+Before you worry too much: we're already checking for this category of wasted work in some form, via the use of a **depth prepass**.
+A depth prepass renders a simplified version of the scene, recording only the depth (distance from the camera) of each object into the 2-dimensional depth buffer.
```

Comment:
Saying "a simplified version of the scene" kinda implies to me that it uses simplified meshes, but that's not what's happening. I think it would be more accurate and less misleading to only mention the depth only part without saying it's a simplified scene. Maybe saying it's a simplified shader?

Reply:
```suggestion
A depth prepass renders the scene with a simplified shader, recording only the depth (distance from the camera) of each object into the 2-dimensional depth buffer.
```